server-spi/src/main/java/org/keycloak/models/RequiredActionProviderModel.java

```diff
@@ -43,6 +45,7 @@ public interface RequiredActionComparator extends Comparator<RequiredActionProvi
     private boolean defaultAction;
     private int priority;
     private Map<String, String> config = new HashMap<>();
+    private boolean configurable;
```

Comment:
Nitpick: This field is not used and can be removed from this class (probably all the changes from this class can be removed from this PR)